recipes/dspsr/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "dspsr" %}
+{% set version = "2022-02-17" %}
+{% set sha256 = "835246c81d78f25b4620cbabf50dfa7ac3a7c1e3145046102a3bb8a54bc5e57b" %}
+
+package:
+    name: "{{ name | lower }}"
+    version: "{{ version | replace('-','.') }}"
+
+source:
+    url: https://sourceforge.net/projects/{{ name }}/files/{{ name }}/{{ version }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    patches:
+        - macos_fmemopen.patch  # [osx]
+        - python_ldflags.patch
+        
+build:
+    number: 0
+    skip: true  # [win or (python_impl == 'pypy')]
```

```suggestion
    skip: true  # [win or (python_impl == 'pypy')]
    run_exports:
      - {{ pin_subpackage('dspsr', exact=True) }}
```
You need a run_export for your package too because you allow people to link to your C++ interface. Using "exact" version pinning because date-based versioning instead of semantic.